celer/PN_logreg.pyx

```diff
@@ -105,16 +105,17 @@ def newton_celer(
         p_obj = primal(LOGREG, alpha, Xw, y, w, weights_pen)
 
         # theta = y * sigmoid(-y * Xw) / alpha
-        create_dual_pt(LOGREG, n_samples, alpha, &theta[0], &Xw[0], &y[0])
+        create_dual_pt(LOGREG, n_samples, &theta[0], &Xw[0], &y[0])
         norm_Xtheta = dnorm_l1(
             is_sparse, theta, X, X_data, X_indices, X_indptr,
             screened, X_mean, weights_pen, center, positive)
 
-        if norm_Xtheta > 1.:
-            tmp = 1. / norm_Xtheta
+        # alpha instead of 1
+        if norm_Xtheta > alpha:
+            tmp = 1. / (norm_Xtheta / alpha)
```

Comment:
probably cleaner: tmp = alpha / norm_Xtheta  (multiple occurrences)

Reply:
Shall we adopt this in all occurrences of ``tmp``?

Reply:
It's true that ``alpha / norm_Xtheta`` is clean. Yet, I think ``1. / (norm_Xtheta / alpha)`` is more readable as it related to the dual equations.

Do we scarify readability for cleanability?

Reply:
Go for alpha/norm everywhere. It is quite readable too.